src/tx/staking/verifyStake.ts

```diff
@@ -43,6 +44,14 @@ export function verifyStakeTx(
     //TODO: NEED to potentially write a custom faster test that avoids regex so we can avoid a regex-dos attack
     success = false
     reason = 'Invalid nominee address in stake coins tx'
+  } else if (
+    nomineeAccount &&
+    nomineeAccount.stakeTimestamp + AccountsStorage.cachedNetworkAccount.current.restakeCooldown > Date.now()
+  ) {
```

Comment:
if nomineeAccount is undefined, should we want to return false or something like that? as it is, this condition will get skipped, as well as the old one on line 88, and it looks like we'll return true